### DISCUSSION TITLE
Is there a way to store keys in globalData and use in HTTP Request

### DISCUSSION THREAD

**Discussion by systemaddict** [AutomaApp/automa#1268] [Q&A]
Hi

I would like to store my sensitive data in globalData instead in the individual nodes. This would make it easier to share workflows (so people don't have to go into several nodes to change/add a key for for example open AI).

But this doesn't seem to work:

![image](https://github.com/AutomaApp/automa/assets/1615733/97a00377-1b72-426b-a9a4-2beaf7728bcb)

Are there any other options?

I guess it's the same issue as this one (that was resolved later): https://github.com/AutomaApp/automa/issues/663

**Comment by SuperTempAccount** [comment]
Hey @systemaddict you can store credentials under `Storage` -> `Credentials` -> `Add credential`.
You can then access the credential in your workflow using `{{ secrets@credentialName }}`.

See https://docs.automa.site/reference/storage.html for details.
